### ISSUE TITLE
Allow Copilot to use existing hosted zone as the base application hosted zone.

### ISSUE THREAD

**Issue by lgarvey** [aws/copilot-cli#4362]
I work for an organisation that has multiple projects/teams. Each team maintains their own AWS account.  We then provide them with a hosted zone off a shared domain, e.g.  theproject.ourorg.digital.   The domain: ourorg.digital lives in a central account, and then theproject.ourorg.digital is set up in the project/team account.

However, copilot then insists on creating an additional hosted zone for the application, so we end up with:

theapplication.theproject.ourorg.digital

e.g 

intranet.intranet.ourorg.digital

It would be useful if instead of copilot creating the application hosted zone, we can tell it to use the base domain hosted zone instead.

**Comment by Lou1415926** [comment]
@lgarvey I wonder if [http.hosted_zone](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#http-hosted-zone) would be an option for you?

That is, instead of creating your app with `copilot app init --domain`, you can:
1. Create and validate your own certificate
2. Import the certificate into your environments under `http.public/private.certificates` or `cdn.certificates`.
3. In your service manifest, specify `http.alias` and `http.hosted_zone`

This way, Copilot will not attempt to create a hosted zone to manage your domains; instead, it will upsert the alias records into the designated hosted zone.

**Comment by lgarvey** [comment]
I think this is the only option tbh. It's how we currently handle our prod domain entries, which differs from the non prod domain.  It's a shame there's not a bit more flexibility because it's pretty cool how it handles DNS/TLS provisioning. 

Thanks.

**Comment by Lou1415926** [comment]
@lgarvey Makes sense to me!  This issue is still a feature request that's not addressed yet, so I will reopen it for us to keep track 👍🏼 .
